scripts/eval_calc_errors.py

```diff
@@ -376,7 +403,7 @@ def calculate_errors_per_image(im_metaData):
                                         R_g=R_g,
                                         t_g=t_g,
                                         depth_im=depth_im,
-                                        K=K,
+                                        K=cam,
```

Comment:
I think vsd doesn't support camera models besides pinhole, would be good to raise an error if anyone tries to evaluate vsd error on Hot3D.

Reply:
Not at the moment indeed, we could implement vsd with the Hand Tracking Toolkit renderer quite easily but this is not a priority for now since vsd in computed for new BOP24 datasets. Will add an error message.

Reply:
Did you add an error message?

Reply:
Not yet, will do soon

Reply:
Added a "supported_error_types" field in dataset_params for early check in `eval_calc_errors.py`